src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -141,7 +141,7 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * The following example demonstrates how the check validates annotations of method parameters,
+ * The following example demonstrates how the check validates annotations of
  * catch parameters, foreach, for-loop variable definitions.
```

Comment:
method parameters and catch parameters are the same token. If you remove 1 you are removing the other.
> Check enforce to locate annotations immediately after documentation block

Catch parameters don't have a documentation block, so it seems they should be treated the same as method parameters.

I don't see any tests affecting catch parameters so it would be good to add some to showcase we won't validate them anymore.
Documentation needs to also remove reference to catch parameters too.
@romani Please confirm I am right.

Reply:
Ah, yes, you are right.
I just assumed they are treated as local variables like for-each and for-loop variables.
Becuase it seemed logic to me and there was no test failing. :-D
Changes are done and pushed.